connectors/utils.py

```diff
@@ -712,7 +712,19 @@ def __init__(self):
         if self.extraction_config is not None:
             self.host = self.extraction_config.get("host", None)
             self.timeout = self.extraction_config.get("timeout", 30)
+            self.headers = {"accept": "application/json"}
             self.chunk_size = self.extraction_config.get("stream_chunk_size", 65536)
+
+            self.use_file_pointers = self.extraction_config.get(
+                "use_file_pointers", False
+            )
+            if self.use_file_pointers:
+                self.volume_dir = self.extraction_config.get(
+                    "shared_volume_dir", "/app/files"
+                )
+            else:
+                self.volume_dir = None
+                self.headers["content-type"] = "application/octet-stream"
```

Comment:
Do we need to set the headers to be `application/json` in the `if` clause to match this? Or does it work without them being set explicitly?

Reply:
It works as is, I think it isn't necessary to set `content-type` to anything for filesend because the other request type is just a PUT with a URI param.

Reply:
🤦 oh duh. For some reason I thought it had a payload, but you're totally right.